numba/tests/test_function_type.py

```diff
@@ -187,7 +187,7 @@ def foo(f, g):
 
         sig = int64(int64)
 
-        for decor in [mk_cfunc_func(sig), mk_wap_func(sig), njit_func,
+        for decor in [mk_cfunc_func(sig), mk_wap_func(sig), # njit_func,
```

Comment:
If this is now unsupported, consider removal instead of commenting it out? (same comment applies throughout).

Reply:
The tests are now cleaned up.